.github/workflows/migrate.yml

```diff
@@ -0,0 +1,49 @@
+name: migrate
+
+on:
+  workflow_dispatch:
+    inputs:
+      build:
+          description: "The data to update"
+          type: choice
+          required: true
+          options:
+              - all
+              - acs
+              - decennial
+      environment:
+          description: "The database to update"
+          type: choice
+          required: true
+          options:
+              - development
+              - staging
+              - production
+
+jobs:
+  update:
+    environment:
+      name: ${{ inputs.environment }}
+    name: Data update...
+    runs-on: ubuntu-22.04
```

Comment:
Can the workflow run on `ubuntu-24.04`/`ubuntu-latest`?
1) April 2027 (Security EOL for ubuntu 22) seems like a long time until it's not
2) Our other runners are on 24 and I find it helpful to keep a common environment across apps, where possible

Reply:
Second this. I'd prefer `ubuntu-24.04` to `ubuntu-latest`.